tests/Directory.Build.props

```diff
@@ -1,13 +1,16 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
+  <Import Project="$(TestsSharedRepoTestingDir)Aspire.Testing.Repo.props" />
```

Comment:
There is a little disconnect between the naming of this - `Aspire.Testing.Repo.*`, and the directory in `tests/Shared/RepoTesting`.

`tests/Shared/TestingRepo` sounds odd though. I could rename the targets to be `Aspire.Repo.Testing.*`. Any preference?

Reply:
Maybe `Aspire.RepoTesting.*`? That would align nicely with the directory name.

Reply:
Fixed.